wagtail/admin/ui/tables/__init__.py

```diff
@@ -243,6 +243,14 @@ def get_cell_context_data(self, instance, parent_context):
         return context
 
 
+class BooleanColumn(Column):
+    """Represents a True/False/None value as a tick/cross/question icon"""
+
+    def get_value(self, instance):
+        value = super().get_value(instance)
+        return display_for_value(value, lambda x: x, boolean=True)
```

Comment:
Not sure if it is a good idea to depend on a Django admin util, but the icons looks pretty good and are easy to use. If we were to add our own icons, we'll have to add the styling etc.

Reply:
While I'm generally OK with us dipping into `django.contrib.admin` for utility functions, unfortunately I don't think we can count on `django.contrib.admin` being present in INSTALLED_APPS, which would be required for the static files to be available.

(Have we really not implemented true/false icons elsewhere...?)

Reply:
We do have check and cross icons, but they are not coloured and will need some stylings. I was worried about the static files too, though. I'll see if we can replace this with our own icons.

Reply:
@gasman I've dropped the commit for `BooleanColumn` to reduce the scope of this PR. This only fixes the falsy value issue and updates the docs. I'll add the `BooleanColumn` separately.

Reply:
Actually, it wasn't so hard to do with our own icons. I've added 86a2d51 and updated the PR description with new screenshots.